get-deps.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+
+#########################################################
+# Checks out site content dependencies needed to generate the full site
+#########################################################
+
+# Check out the events repo as a direct subdir
+#git clone --depth=1 https://github.com/Terracotta-OSS/terracotta.org-site-events.git events
+
+# Check out a single copy of quartz to use below:
+git clone https://github.com/quartz-scheduler/quartz.git __reference
+pushd __reference
+
+# Copy latest version of quartz docs to a subdir:
+git checkout master
+cp -rf docs ../documentation/2.4.0-SNAPSHOT
+
+# Copy 2.3.x version of quartz docs to a subdir:
+git checkout quartz-2.3.x
```

Comment:
It may be worthwhile to keep the get-deps.sh plus [the ruby script](https://github.com/ehcache/ehcache.org-site/blob/master/run_cmd_each_javadoc_version.rb) as in ehcache3, because this eliminates duplication of configuration, making the jekyll [_config.yml](https://github.com/ehcache/ehcache.org-site/blob/master/_config.yml) the authoritative source for external git prerequisites.  

This script would then not need to be changed as branches are added.

Up to you though.

Reply:
yeah, I am aware of those ruby script. For now though, I just want to get the build and site going. Let me get a hang of all of these flow, then we can automate it further.